tools/depends/configure.ac

```diff
@@ -437,6 +467,7 @@ case $use_platform in
   raspberry-pi3)
      target_platform=raspberry-pi
      use_cpu=cortex-a53
+     use_cmake_cpu=aarch64
```

Comment:
Here and above becomes related to Raspberry the correct cmake cpu set

Reply:
Are you sure this is correct?

Reply:
About them https://en.wikipedia.org/wiki/ARM_Cortex-A53 is it only 64bit. Is it possible to use this a 32bit?

Reply:
Yes the cpu might be 64 Bit. Should this value really reflect the architecture of the Cortex-A53 processor, or wouldn't be the targeting architecture of the produced binaries the only correct value?

Reply:
Have looked a little more so more would be wrong. The `Cortex-A53` is used from Version Mod. B v1.2 (Cortex-A7, 32bit), the older have (ARM1176JZF-S, 32bit) and the new have Cortex-A53 64bit. A bit hard mix in one Version

Reply:
They use mostly only `armv7-a` the 64bit is available, but mostly still not supported on Raspberry.